test/module/irohad/ametsuchi/postgres_executor_test.cpp

```diff
@@ -2611,7 +2611,53 @@ namespace iroha {
       auto kv = sql_query->getAccountDetail(account_id);
       ASSERT_TRUE(kv);
       ASSERT_EQ(kv.get(), R"({"id@domain": {"key": "value"}})");
-    }
 
-  }  // namespace ametsuchi
-}  // namespace iroha
+      class SetSettingValueTest : public CommandExecutorTest {
+        void SetUp() override {
```

Comment:
Is this override required? It does not modify the behavior.

Reply:
Fixed